package.json

```
@@ -1289,6 +1301,24 @@
             "type": "string",
             "default": ";",
             "markdownDescription": "%cosmosdb.configuration.csv.separator%"
+          },
+          "cosmosDB.shell.path": {
+            "type": [
+              "string",
+              "null"
+            ],
+            "description": "%cosmosdb.configuration.cosmosDB.shell.path%",
+            "default": null
+          },
+          "cosmosDB.shell.MCP.enabled": {
+            "type": "boolean",
+            "default": true,
```

```suggestion
            "default": false,
```
Let's turn this off by default